readme.md

```diff
@@ -50,6 +50,10 @@
 - [Minecraft Forums](https://www.minecraftforum.net/) - Official community forums, a great place to talk about the game.
 - [Planet Minecraft](https://www.planetminecraft.com/forums/) - Creative community fansite.
 
+### Tools
```

Comment:
I think it should be "Documentations", not "Tools"?

Reply:
Hmm idk, it feels not kinda it for it. Maybe "Indexes"?
if not "Indexes" then its Documentation

Reply:
yeah, tools isn't right, either documentation or indexes is good.